### ISSUE TITLE
Installing trigger fails on custom named tables

### ISSUE THREAD

**Issue by gprzyby** [Opus10/django-pgtrigger#55]
Recently I've faced a problem in my project when running `./manage.py pgtrigger install`
```
pgtrigger: Installing set_paid_date trigger for Order table on default database.
Traceback (most recent call last):
  File "$PROJECT_HOME/lib/python3.10/site-packages/django/db/backends/utils.py", line 87, in _execute
    return self.cursor.execute(sql)
psycopg2.errors.SyntaxError: syntax error at or near "Order"
LINE 4:                     BEFORE UPDATE OR INSERT ON Order
```

Problematic code:
```
@pgtrigger.register(
    FixedTrigger(
        name="set_paid_date",
        operation=pgtrigger.Update | pgtrigger.Insert,
        when=pgtrigger.Before,
        condition=pgtrigger.Q(new__status="PAID"),
        func="NEW.payment_date = now(); RETURN NEW;",
    ),
)
class Order(models.Model):
    customer = models.ForeignKey(
        "Customer", related_name="orders", on_delete=models.CASCADE
    )
    create_date = models.DateTimeField(auto_now=True)
    payment_date = models.DateTimeField(null=True)
    shipment_date = models.DateTimeField(null=True)
    status = models.CharField(
        max_length=45, choices=OrderStatus.choices, default=OrderStatus.CREATED
    )
    realization_date = models.DateTimeField(null=True)
    products = models.ManyToManyField(
        "Product",
        through="OrderProduct",
        through_fields=("order", "product"),
        related_name="orders",
    )

    class Meta:
        db_table = "Order"
```

It was kind of curious why excepiton occures, so I dive into library code, and found solution :D 
Imo there is an error in custom named tables. In docs I haven't found similar example with custom named table(using custom named tables is not often practice). There is an name conflict in sql order statement and "order" table name; 

How to fix:
In function `pgtriggers.Trigger.render_trigger` SQL query needs double quoted table name. After this fix also faced same error in function `pgtriggers.Trigger.render_comment` and double-quoting table name fixed problem:

Fixed Trigger class:
```
from pgtrigger import Trigger


class FixedTrigger(Trigger):
    def render_trigger(self, model):
        """Renders the trigger declaration SQL statement"""
        table = model._meta.db_table
        pgid = self.get_pgid(model)
        return f'''
            DO $$ BEGIN
                CREATE TRIGGER {pgid}
                    {self.when} {self.operation} ON "{table}"
                    {self.referencing or ''}
                    FOR EACH {self.level} {self.render_condition(model)}
                    EXECUTE PROCEDURE {pgid}();
            EXCEPTION
                -- Ignore issues if the trigger already exists
                WHEN duplicate_object THEN null;
            END $$;
        '''

    def render_comment(self, model):
        """Renders the trigger commment SQL statement

        pgtrigger comments the hash of the trigger in order for us to
        determine if the trigger definition has changed
        """
        pgid = self.get_pgid(model)
        hash = self.get_hash(model)
        table = model._meta.db_table
        return f'COMMENT ON TRIGGER {pgid} ON "{table}" IS \'{hash}\''

```

**Comment by gprzyby** [comment]
I've forgot about using version: 2.4.1

**Comment by wesleykendall** [comment]
@gprzyby FYI this issue is fixed as of #60
